packages/app/src/debug/DebugContainer.vue

```diff
@@ -1,16 +1,30 @@
 <template>
-  <div>
+  <div class="h-full">
     <DebugError
       v-if="showError"
     />
     <div
-      v-else-if="loginConnectStore.user.isLoggedIn && loginConnectStore.project.isProjectConnected && run"
+      v-else-if="loginConnectStore.user.isLoggedIn && loginConnectStore.project.isProjectConnected && run && run.status"
+      class="flex flex-col h-full"
     >
       <DebugPageHeader
         :gql="run"
         :commits-ahead="0"
       />
+      <DebugPageDetails
+        :status="run.status"
+        :specs="run.specs"
+        :cancelled-at="run.cancelledAt"
+        :cancelled-by-full-name="run.cancelledBy?.fullName"
+        :cancelled-by-email="run.cancelledBy?.email"
+        :is-hidden-by-usage-limits="run.isHiddenByUsageLimits"
+        :over-limit-action-type="run.overLimitActionType"
+        :over-limit-action-url="run.overLimitActionUrl"
+        :ci="run.ci"
+        :errors="run.errors"
+      />
       <DebugSpecList
+        v-if="run.totalFailed && ['ERRORED','CANCELLED','TIMEDOUT', 'FAILED'].includes(run.status)"
```

Comment:
I would extract this into a guard function to keep the template simpler, but just a personal preference

```
function shouldDisplaySpecsList() {
  ...
}
```